lib/ddtrace/profiling/preload.rb

```diff
@@ -1,4 +1,3 @@
 # typed: strict
-require 'ddtrace'
 
```

Comment:
This one is also required, as it gets used by `bin/ddtraceb` to load the gem and start profiling immediately (and similarly to others, does not get required directly or indirectly from `ddtrace.rb`)